woocommerce/woocommerce-ios
# WCPay setup instructions screen

Part of: #10571 


## Description
Adds a setup instructions screen for Woo Payments plugin setup onboarding task.

## Testing instructions

Free trial
1. Login into a free trial store
2. Tap on the "Get paid" onboarding task
3. Validate that the new WCPay setup instructions screen is displayed

Create a JN site
1. Login into a JN store
2. Tap on the "Get paid" onboarding task
3. Validate that you don't see the new WCPay setup instructions screen

## Screenshots
| Light | Dark |
|--------|--------|
| ![Simulator Screen Shot - iPhone 14 Pro - 2023-08-30 at 14 10 28](https://github.com/woocommerce/woocommerce-ios/assets/524475/987af4c3-f40d-4329-ac69-6e0dd3dbec50) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-08-30 at 14 36 50](https://github.com/woocommerce/woocommerce-ios/assets/524475/17125ac3-fa99-4b4f-99b4-44ca0bc69fd0) |

Flow
| Payments task | Woo Payments task |
|--------|--------|
| ![Simulator Screen Recording - iPhone 14 Pro - 2023-08-30 at 14 11 17](https://github.com/woocommerce/woocommerce-ios/assets/524475/52499941-4d0b-473d-9322-13d58a428372) | ![Simulator Screen Recording - iPhone 14 Pro - 2023-08-30 at 14 10 47](https://github.com/woocommerce/woocommerce-ios/assets/524475/0513771f-9da7-452a-ab83-34c8a9f7aa33) | 



---
